libkbfs/file_data_test.go

```diff
@@ -405,6 +411,11 @@ func testFileDataLevelExistingBlocks(t *testing.T, fd *fileData,
 		prevChildren = level
 		numLevels++
 	}
+
+	if numLevels > 1 {
+		fd.file.path[len(fd.file.path)-1].DirectType = IndirectBlock
```

Comment:
The `path` represents the set of directories leading up the file; the tail pointer of the file is actually the root block.  (Same as `fs.rootBlockPointer()`.)  So if there's more than one level in the file, this sets the root block to be indirect.